common/constants.js

```diff
@@ -148,6 +148,10 @@ module.exports = Object.freeze({
     HUMAN_INTERVAL: 'human-readable-interval',
     CRON_EXPRESSION: 'cron_expression'
   },
+  CONFIG: {
+    DISABLE_SCHEDULED_UPDATE_CONFIG_PREFIX: 'disable_scheduled_update_',
+    RESOURCE_NAME: 'sfconfig'
```

Comment:
can you name this const `RESOURCE_NAME` to `SF_CONFIG`? because down the line we might add another config & then we would have to name that constant something differently. Also, the following reads better `CONST.CONFIG.SF_CONFIG` as you are saying fetch a config param from SF_CONFIG.